exercises/twelve-days/canonical-data.json

```diff
@@ -0,0 +1,121 @@
+{
+  "exercise": "twelve-days",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "verse",
+      "cases": [
+        {
+          "description": "first day a partridge in a pear tree",
+          "property": "recite",
+          "start verse": 1,
+          "end verse": 1,
+          "expected": "On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree."
+        },
+        {
+          "description": "second day two turtle doves",
+          "property": "recite",
+          "start verse": 2,
+          "end verse": 2,
+          "expected": "On the second day of Christmas my true love gave to me, two Turtle Doves, and a Partridge in a Pear Tree."
```

Comment:
Could you encode the expected value as an array of strings? See #983 and #973. And do the same for the other "expected" properties?